_includes/footer.html

```diff
@@ -4,14 +4,18 @@
   CC BY NC 4.0</a>
   </p>
 
-  <p>Copyright 2009-2015 Software Freedom Conservancy, Sugar Labs®</p>
+  <p id="copy"></p>
+<script>
+  document.getElementById("copy").innerHTML="Copyright &copy; 2009-"+new Date().getFullYear()+ " Software Freedom Conservancy, Sugar Labs&reg;";
+</script>
 
   <!-- FIXME: Update/remove pages -->
   <p>Connect with us on
     <a href="https://socialhelp.sugarlabs.org">Social Help</a> or
     <a href="https://www.facebook.com/SugarLabs-187845102582/timeline/">Facebook</a> or
-    <a href="https://github.com/sugarlabs">Github</a>
+    <a href="https://github.com/sugarlabs">GitHub</a>
```

Comment:
Fixed in https://github.com/sugarlabs/www-sugarlabs/commit/a4d6c6f38c1bfdf23a4d2f49eedba2bd6c8bd594